test/template.html

```diff
@@ -1,6 +1,3 @@
-<meta content="authenticity_token" name="csrf-param" />
```

Comment:
In ff8ba39 I replaced the dollar selector with a `document.head.querySelector`, therefor the csrf meta tags actually have to exist inside the head. I _thought_ this would not be a problem, but I now see that [`rails-ujs` uses a general `document.querySelctor`](https://github.com/rails/rails-ujs/blob/8a8a9c867e7a5c6ad0ab08abd5369ee9989464d7/src/utils/csrf.coffee) so that the csrf meta tags can appear outside of the head. Will revert this change.